README.rst

```diff
@@ -13,18 +13,18 @@ implementation.
 This gap is now too high and can lead you to unexpected side effects and bugs
 in your applications.
 
-**Eventlet is now switching to legacy mode**. **Only maintenance for stability
-and bug fixing** will be provided. **No new features will be accepted**, except
-those related to the asyncio migration. **Usage in new projects are
-discouraged**. **Our goal is to plan the retirement of eventlet** and to give
-you ways to move away from eventlet.
+Eventlet now follows a new maintenance policy. **Only maintenance for
+stability and bug fixing** will be provided. **No new features will be
+accepted**, except those related to the asyncio migration. **Usages in new
```

Comment:
Maybe "Using eventlet in new projects is discouraged."?

Reply:
let's update this sentence later if really needed.